loadtest/beats-yaml-generator.py

```diff
@@ -0,0 +1,21 @@
+import os
```

Comment:
Add a comment, possibly including an example of the `kubectl` output you are trying to parse.